packages/peregrine/lib/util/preserveQueryParams.js

```diff
@@ -1,10 +1,11 @@
+import { persistentQueries } from './persistentQueries';
 /**
  * Function takes location object, filters through it and creates
  * new query params with preserved values from the array.
  * @param {object} location
  * @param {array} queries
  */
-export const preserveQueryParams = (location, queries) => {
+export const preserveQueryParams = (location, queries = persistentQueries) => {
```

Comment:
Not sure if this is correct but it was the easiest way to move this to peregrine, for now.

We use this in both `peregrine` (in asyncAction for catalog) and `venia-ui` (in filterFooter). I guess, if someone wanted to use this utility, they would need to know _what_ the defaults were at least. 

TODO: Update the jsdoc comment.